### ISSUE TITLE
Short-form command options should have their long-form listed

### ISSUE THREAD

**Issue by ThomasN-Saab** [NuGet/docs.microsoft.com-nuget#999]
I assume that the "-a" used in the examples is the option "AssemblyPath", but that isn't written anywhere. If the options have short forms those should be listed! 

Also, must I be in the same folder as the .csproj file? Can I not reference it with an address? What if I have multiple .csproj files in the same folder? Is there a way to designate an output path? If not then these are pretty serious shortcomings.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ddd89a40-99ee-25da-3a4b-a670f8954acd
* Version Independent ID: 61aad6bb-4488-810d-476b-211a3ee9cb0a
* Content: [NuGet CLI spec command](https://docs.microsoft.com/en-us/nuget/tools/cli-ref-spec)
* Content Source: [docs/tools/cli-ref-spec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/tools/cli-ref-spec.md)
* Product: **nuget**
* GitHub Login: @karann-msft
* Microsoft Alias: **karann**

**Comment by shaulbehr** [comment]
Agreed. This documentation has almost no useful information.

**Comment by karann-msft** [comment]
@ThomasN-Saab 
> I assume that the "-a" used in the examples is the option "AssemblyPath", but that isn't written anywhere. If the options have short forms those should be listed!

Have fixed that with this commit - https://github.com/NuGet/docs.microsoft.com-nuget/commit/ce561d8183497c2e038c3cac12babc3820d6f197

> Also, must I be in the same folder as the .csproj file? 

No, it's not a mandate.

> Can I not reference it with an address?

If you mean relative path, then yes you can.

> What if I have multiple .csproj files in the same folder?

Do you have such a solution structure where multiple csproj files are sitting right next to each other? How and why did you get into that state?

Is there a way to designate an output path?
> No there isn't.

@shaulbehr we would appreciate your feedback on what is missing and how we can make the info more useful?

**Comment by ThomasN-Saab** [comment]
So you're saying that there are no shortforms for the options then?

If I don't have to be in the same folder as the .csproj file, why doesn't the documentation say how I can reference one in another folder? -AssemblyPath seems to be for assemblies and PackageID seems to be something else entirely.

Having multiple .csproj files in the same folder is not unusual. I've seen it in several open source projects; some of which I'm trying to create .nuspec files for. However if I can, as you say, reference a .csproj file with a relative path then that shouldn't be an issue (as that would imply specifying a specific .csproj file), but the documentation doesn't show how.

Basically, how would I correctly write the following command?
``nuget spec ..\myProject.csproj -Force -OutputFolder ..\NuGet\``
You did say that I can't designate an output path, but I added that to the command anyway just to show what exactly it is I want to do. The fact that option is missing is unfortunate as it would be very helpful. I guess I'll have to contact the NuGet team with a feature request.

**Comment by ThomasN-Saab** [comment]
This issue was closed. Does that mean that the short forms of the options will be added to the documentation?

**Comment by CamSoper** [comment]
@ThomasN-Saab I'm sorry, I was triaging the entire issue list last night and I thought this one was resolved since you created a feature request in the other repo. Upon re-review, it looks to me as if I misunderstood. It appears as if that request was not entirely related to the docs request.  My mistake!  Sorry about that.  Reopening and assigning priority.

**Comment by karann-msft** [comment]
Docs day - scan all docs for short-forms and fix.
